### PR TITLE
gets also ubuntu version

### PR DESCRIPTION
**Pull request by yoavrotems**
There are differences between Ubuntu 16 and Ubuntu 18 features so the system needs to know the specific version

### CODE REVIEW

File: utils.go [hunk 1]
```diff
@@ -104,11 +104,16 @@ func GetLSM() (lsm string, err error) {
 func getPlatformVersion(output, platform string) string {
 	flagRe := regexp.MustCompile("version_id" + `=([^ \n]*)`)
 	vals := flagRe.FindStringSubmatch(output)
-
+	if vals == nil {
+		flagRe := regexp.MustCompile("version" + `=([^ \n]*)`)
+		vals = flagRe.FindStringSubmatch(output)
```
Comment (lizrice) [inline utils.go]: I think you could match both options with one regex, something like `version[_id]*=([^ \n]*)` (within backtick quotes). Try it out at regex101.com to check.

Comment (yoavrotems) [reply utils.go]: Less is better :) 
would make the change :)

**Comment by lizrice** [comment]
@yoavrotems you don't need to start a new PR when you make an update, it's better to push your changes to the existing one.

**Comment by lizrice** [comment]
@yoavrotems you're still planning to update the regex rather than having two of them, right?

**Pull request merged**